code-challenges.test.js

```diff
@@ -12,43 +12,105 @@
+// .shift removes the first element of an array and returns that removed element
+// randomizer is Math.floor(Math.random() * array.length)
+// going to need more time as my og idea did not work.. going to work on this past monday...
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
-
+// describe("remRandArr", () => {
+
+//     // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+//     it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+  
+//       //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+//       expect(remRandArr(colors1)).toEqual(["yellow", "blue", "pink", "green"])
+//       expect(remRandArr(colors2)).toEqual(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"])
+//     })
+//   })
 var colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+// // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+// console.log(colors1.pop(colors1[0]))
 
 
 // b) Create the function that makes the test pass.
-
-
-
+// this would work in a perfect world 
+
+    // remRandArr = (arr) => {
+    //     arr.reverse().pop()
+    //     const newArr = []
+    //     while (newArr.length < arr.length){
+    //         if (newArr.includes(arr[Math.floor(Math.random() * arr.length)])){
+    //             newArr.push(arr[Math.floor(Math.random() * arr.length)])
+    //         }
+    //     }
+    //     return console.log(newArr)
+    // }
+    
+
+    // remRandArr(colors1)
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("minMax", () => {
 
-var nums1 = [3, 56, 90, -8, 0, 23, 6]
-// Expected output: [-8, 90]
-var nums2 = [109, 5, 9, 67, 8, 24]
-// Expected output: [5, 109]
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it("takes an array of numbers and returns an array of the minimum and maximum numbers in that order.", () => {
+  
+      //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+      // var nums1 = [3, 56, 90, -8, 0, 23, 6]
```

Comment:
I think having these test variables commented out is affecting the ability of your test to run.